From f46021dd741298311c730e611ad401d9a64fc23d Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Sun, 8 May 2016 18:30:56 +0300
Subject: [PATCH 380/538] fix: i2c: dts: update Armada-7k/8k to use I2C-Bridge
 driver

Issue found when U-Boot start use I2C-bridge driver and linux keep
using old I2C unit
This patch update the Armada-7k/8k SoC to use I2C-Bridge driver
SYSTEMSW-2420

Change-Id: I0824d0650ce2cc8c0c24fbe5cab8c1bbd5a33f88
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29607
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 2 +-
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index ab08af5..b52d265 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -213,7 +213,7 @@
 			};
 
 			i2c0: i2c@511000 {
-				compatible = "marvell,mv64xxx-i2c";
+				compatible = "marvell,mv78230-i2c";
 				reg = <0x511000 0x20>;
 				#address-cells = <1>;
 				#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 3a2ee1d..a9d21b4 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -119,7 +119,7 @@ spi@700680 {
 };
 
 i2c1: i2c@701000 {
-	compatible = "marvell,mv64xxx-i2c";
+	compatible = "marvell,mv78230-i2c";
 	reg = <0x701000 0x20>;
 	#address-cells = <1>;
 	#size-cells = <0>;
@@ -130,7 +130,7 @@ i2c1: i2c@701000 {
 };
 
 i2c2: i2c@701100 {
-	compatible = "marvell,mv64xxx-i2c";
+	compatible = "marvell,mv78230-i2c";
 	reg = <0x701100 0x20>;
 	#address-cells = <1>;
 	#size-cells = <0>;
-- 
1.9.1

